commands/serverless.go

```diff
@@ -58,12 +58,12 @@ func Serverless() *Command {
 	cmd := &Command{
 		Command: &cobra.Command{
 			Use:   "serverless",
-			Short: "Develop and test serverless functions",
-			Long: `The ` + "`" + `doctl serverless` + "`" + ` commands provide an environment for developing and testing serverless functions.
-One or more local file system areas are employed, along with a 'functions namespace' in the cloud.
+			Short: "Develop, test, and deploy serverless functions",
+			Long: `The ` + "`" + `doctl serverless` + "`" + ` commands provide an environment for developing, testing, and deploying serverless functions.
+One or more local file system areas are employed, along with one or more 'functions namespaces' in the cloud.
```

Comment:
In addition to mentioning multiple namespaces here, the language suggesting that `doctl serverless` can only be used for testing and not final deployment is relaxed, since we do now support using the result of `doctl serverless deploy` in production.